docs/project/2024/vlad_andrei.dumitru/index.md

```diff
@@ -104,6 +109,9 @@ The format is
 | Library | Description | Usage |
 |---------|-------------|-------|
 | [micro-rand](https://crates.io/crates/micro_rand) | A tiny, no STD library for generating (pseudo) random numbers. | Used to generate the winning number |
+| [LCD1602](https://github.com/mihai1803/1602lcd) | A library for interfacing with the LCD1602 module. | Used to display game information |
+
+For the software part, the only library I used was micro-rand, which is a tiny, no STD library for generating (pseudo) random numbers. This library was used to generate the winning number in the game. Otherwise, the game logic works on basic asynchronous tasks and loops from Embassy.
```

Comment:
Are you sure you did not use any other libraries ? rp-hal, embedded-hal, embassy?

Reply:
Please mention all the libraries that you used except the ones that come with `embassy-rs` or are its dependencies.